frontend/src/layouts/AppLayout.jsx

```diff
@@ -1,21 +1,42 @@
-import { Outlet, useNavigate } from "react-router-dom";
+import { Outlet, useLocation, useNavigate } from "react-router-dom";
+import { LayoutItem } from "../components/LayoutItem";
 import { logout } from "../api/auth";
+import css from "././AppLayout.module.css";
 
 export default function AppLayout() {
   const navigate = useNavigate();
 
+  const { pathname } = useLocation();
+
+  console.log(pathname);
+
   const handleLogout = async () => {
     const { data } = await logout();
     if (data.success) navigate("/");
   };
 
   return (
-    <div>
-      App layout hi there
-      <button onClick={handleLogout} type="button">
-        Logout
-      </button>
-      <Outlet />
+    <div className={css.backdrop}>
+      <div className={css.container}>
+        <nav className={css.menu}>
+          <LayoutItem isActive={pathname.includes("ballots")} route="/app/ballots" title="My Ballots" inTop />
+          <LayoutItem
+            isActive={pathname.includes("available-votings")}
+            route="/app/available-votings"
+            title="Available Votings"
+          />
+          <LayoutItem isActive={pathname.includes("ended-votings")} route="/app/ended-votings" title="Ended Votings" />
+          <LayoutItem
+            isActive={pathname.includes("create-voting")}
```

Comment:
I think it's appropriate to check the isActive logic inside the component. It could be something like this
```js
const { pathname } = useLocation();
const isActive = pathname.startsWith(route);
```